gms/gossiper.cc

```diff
@@ -387,8 +389,9 @@ future<> gossiper::do_send_ack2_msg(msg_addr from, utils::chunked_vector<gossip_
 // - on_restart callbacks
 // - on_join callbacks
 // - on_alive callbacks
-future<> gossiper::handle_ack2_msg(msg_addr from, gossip_digest_ack2 msg) {
-    logger.trace("handle_ack2_msg():msg={}", msg);
+future<> gossiper::handle_ack2_msg(const rpc::client_info& cinfo, gossip_digest_ack2 msg) {
```

Comment:
> In later patch you call `get_endpoint_id(cinfo)`. Maybe we could do that before `background_msg`, and pass `endpoint_id` to `background_msg` call instead of `client_info`?

Sounds like a good idea.  Will look into it.